lib/iris/tests/unit/fileformats/netcdf/test_Saver.py

```diff
@@ -472,6 +472,33 @@ def test_masked_byte_fill_value_passed(self):
                 pass
 
 
+class Test_cf_valid_var_name(tests.IrisTest):
+    def test_no_replacement(self):
+        self.assertEqual(Saver.cf_valid_var_name('valid_Nam3'),
+                         'valid_Nam3')
+
+    def test_special_chars(self):
+        self.assertEqual(Saver.cf_valid_var_name('inv?alid'),
+                         'inv_alid')
+
+    def test_leading_underscore(self):
+        self.assertEqual(Saver.cf_valid_var_name('_invalid'),
+                         'var__invalid')
```

Comment:
Do we care to have `var_invalid` instead of `var__invalid`?

Reply:
I don't. Do you?